supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiConfiguration.scala

```diff
@@ -8,7 +8,6 @@ import cromwell.core.BackendDockerConfiguration
 import net.ceedubs.ficus.Ficus._
 import spray.json._
 
-
```

Comment:
ToL: I'd recommend reverting so that this file is removed from the commit, but it doesn't affect running the code so it _ok_ to merge as is.